content/en/docs/concepts/configuration/secret.md

```diff
@@ -206,81 +252,22 @@ spec:
       defaultMode: 0400
 ```
 
-Then, the secret will be mounted on `/etc/foo` and all the files created by the
-secret volume mount will have permission `0400`.
-
-Note that the JSON spec doesn't support octal notation, so use the value 256 for
-0400 permissions. If you use YAML instead of JSON for the Pod, you can use octal
-notation to specify permissions in a more natural way.
-
-Note if you `kubectl exec` into the Pod, you need to follow the symlink to find
-the expected file mode. For example,
-
-Check the secrets file mode on the pod.
-```
-kubectl exec mypod -it sh
-
-cd /etc/foo
-ls -l
-```
+The secret is mounted on `/etc/foo`; all the files created by the
+secret volume mount have permission `0400`.
```

Comment:
Where are these content now?

Reply:
Gone. This page is about Secrets and is quite long. I propose not telling readers here about how to represent POSIX file modes in JSON, because it's several steps removed from answering the question “In Kubernetes, what is a Secret?”

Reply:
I agree this is not the right place to mention POSIX file modes. Somewhere we still need to call this out because it is something that may get people trapped.

Reply:
Could that be on a (future) task page? I could log an issue to to that migration etc.

Reply:
Logging an issue sounds an option, but the rule of thumb here is do not aimlessly drop useful information.

Reply:
I've revised the wording to include the cautionary message, less verbosely.